### ISSUE TITLE
low accuracy on high dimensional data

### ISSUE THREAD

**Issue by Salehoof** [Xtra-Computing/thundersvm#236]
Hello there
I'm trying to use thundersvm on  [20 newsgroups dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_20newsgroups.html). When I use the

**Comment by QinbinLi** [comment]
Hi @Salehoof ,

You can tune the parameters of SVC to get a good model. For example, if you try`svm.SVC(gamma=0.5, C=100)`, the accuracy of ThunderSVM is 0.9.
